Build/Check cli commands throw Error: Cannot find module 'jsx-email'

<!--
  ⚡️ katchow! We 💛 issues.

  🚨 Your issue will be CLOSED if:
     - This template is removed
     - Parts of this template are removed

  👉🏽 Need help or tech support? Please don't open an issue!
  Head to https://discord.gg/FywZN57mTg or start a Discussion https://github.com/shellscape/jsx-email/discussions

  ❤️ JSX email? Please consider sponsoring our maintainers or telling others about us!
-->

- Component or Package Name: jsx-email
- Component or Package Version: 1.7.2
- `@jsxp-email/cli` Version?:
- Operating System (or Browser): Chrome
- Node Version: 20.10.0
- Link to reproduction _(⚠️ read below)_: https://stackblitz.com/edit/jsx-email-repro-vzgqmc

<!--
  🚨 Issues WITHOUT a valid reproduction WILL BE CLOSED!

  Please provide one by:
  1. Using the StackBlitz reproduction template at https://stackblitz.com/fork/jsx-email-repro
  2. Provide a minimal repository link (Read https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff for instructions).
     Please use NPM for installing dependencies!
     These may take more time to triage than the other options.

  ⚠️ ZIP Files are unsafe and maintainers will NOT download them.
-->

### Expected Behavior

Running **email build ./templates/Reproduction.tsx** should successfully build the template.

Running **email check ./templates/Reproduction.tsx** should successfully run the compatibility check.

### Actual Behavior

Throws and error instead: **Error: Cannot find module 'jsx-email'**

### Additional Information

The behaviour is the same in local Windows environment as in repro link.

<!--
  Most issues can be expressed or demonstrated through StackBlitz or a repository.
  However, the situation may arise where some small code snippets also need to
  be provided. In that situation, please add your code below using
  Fenced Code Blocks (https://help.github.com/articles/creating-and-highlighting-code-blocks/)
-->

--- a/packages/jsx-email/src/cli/commands/build.ts
+++ b/packages/jsx-email/src/cli/commands/build.ts
@@ -121,7 +121,6 @@ const compile = async (files: string[], outDir: string) => {
   await esbuild.build({
     bundle: true,
     entryPoints: files,
-    external: ['jsx-email', 'react'],
     jsx: 'automatic',
     logLevel: 'error',
     outdir: outDir,

